lib/typescript/translations/translations.ts

```diff
@@ -1334,6 +1343,9 @@ const resources = {
       updating: 'Actualizar...',
       webhookCapslock: 'WEBHOOK',
       allEvents: 'TODOS LOS EVENTOS',
+      subscribe: 'Suscripción',
+      noWebhooks: 'No se han encontrado webhooks',
+      noWebhooksText: 'No tiene instalado ningún Webhooks, por favor, ',
     },
   },
```

Comment:
I would remove the `please`  at the end for FR and Spanish (and replace it with a dash), as the sentence sounds strange with it